### DISCUSSION TITLE
[C]About the problem that the printf function before the scanf function is executed will not be displayed in the terminal

### DISCUSSION THREAD

**Discussion by 1185595019** [thebaselab/codeapp#957] [Q&A]
![image](https://github.com/thebaselab/codeapp/assets/61859567/e6ec247c-82fc-4f2b-a6fc-e1cc24abeb8e)

**Comment by namsmith1** [comment]
Having the same problem too, really appriciate a help.

**Comment by bummoblizard** [comment]
Add this to the beginning of your program: `setbuf(stdout, NULL);`
